src/common/liboptparse/optparse.c

```diff
@@ -736,7 +736,7 @@ optparse_t *optparse_create (const char *prog)
     p->option_width = 25;
     p->option_index = -1;
     p->posixly_correct = 1;
-    p->sorted = 1;
+    p->sorted = 0;
```

Comment:
probably just add to the commit something about tests being updated

Reply:
Ah, yeah, meant to do that. Thanks for catching it!

Reply:
Ok,  I added the missing note in the commit message. Thanks.